SelfDriveEnv.py

```diff
@@ -351,54 +365,56 @@ def __init__(self, x, y, num_sensors, reward_func=None):
                 Initial topleft vertical pixel position of the car
             num_sensors: (int)
         """
+        self.config = config
```

Comment:
Is there a reason to have Car have a config while all its necessary information is already set through the other lines in __init__? *Doesn't affect anything, just two lines where it is used in the reset where you could just use the config dictionary because it's imported for the entire file.

Reply:
I do this so the user can specify an updated config. The one in config.py is used as a default. The flow will make more sense when I update the testing docs

Reply:
Ahhh okok gotcha.